pages/1.13/security/oss/managing-authentication/troubleshooting-auth/index.md

```diff
@@ -11,10 +11,10 @@ During testing, we have observed issues with loading the DC/OS UI login page whe
 
 ## Debugging
 
-To debug authentication problems, check the Admin Router and `dcos-oauth` logs on the masters using the following commands.
+To debug authentication problems, check the Admin Router and `dcos-bouncer` logs on the masters using the following commands.
```

Comment:
It probably helps to distinguish login and authentication.

In open DC/OS, Admin Router is the _only_ authenticator in the system. That is, for debugging authentication problems it makes sense to inspect that log.

Bouncer, though, is the only entity emitting authentication tokens. For login problems it makes sense to look into the Bouncer logs.